docs/docs/config-overview.md

```diff
@@ -56,6 +56,13 @@ oh-my-posh --config sample.json --shell uni
 If all goes according to plan, you should see the prompt being printed out on the line below. In case you see a lot of
 boxes with question marks, set up your terminal to use a [supported font][font] before continuing.
 
+:::tip
+The `--config` flag can accept either a local filepath or remotely hosted JSON.
```

Comment:
JSON -> config as it could also be YAML/TOML

Reply:
Of these two, which do you prefer?

``` mdx
The `--config` flag can accept either a local filepath or remotely hosted JSON/YAML/TOML.
```
or 
``` mdx
The `--config` flag can accept either a local filepath or a remotely hosted config file.
```

Reply:
the second :-)